static/app/views/performance/landing/widgets/components/accordion.tsx

```diff
@@ -0,0 +1,80 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import DropdownButton from 'sentry/components/dropdownButton';
+import space from 'sentry/styles/space';
+
+type Props = {
+  content: React.ReactNode;
+  items: (() => React.ReactNode)[];
+  selectedIndex: number;
+  setSelectedIndex: (index: number) => void;
+  radioColor?: string;
+};
+
+export default function Accordion(props: Props) {
+  return (
+    <AccordionContainer>
+      {props.items.map((item, index) => (
+        <AccordionItem
+          {...props}
+          isSelected={index === props.selectedIndex}
+          currentIndex={index}
+          key={index}
+          content={props.content}
+        >
+          {item()}
+        </AccordionItem>
+      ))}
+    </AccordionContainer>
+  );
+}
+
+function AccordionItem({
+  isSelected,
+  currentIndex: index,
+  children,
+  setSelectedIndex,
+  content,
+}: {
+  children: React.ReactNode;
+  content: React.ReactNode;
+  currentIndex: number;
+  isSelected: boolean;
+} & Props) {
+  return (
+    <Fragment>
+      <ListItemContainer>
+        {children}
+        <StyledDropdownButton
+          size="zero"
+          borderless
+          onClick={() => setSelectedIndex(index)}
+          isOpen={isSelected}
+        />
+      </ListItemContainer>
+      <StyledContentContainer>{isSelected && content}</StyledContentContainer>
+    </Fragment>
+  );
+}
+
+const AccordionContainer = styled('div')`
```

Comment:
yeah semantically speaking it probably should. I didn't want to bother dealing with the default `ul` styling but since you're calling me out I'll use `ul` 😂

Reply:
The advantage here is that when you're using react testing library you can specifically do `getByRole('list')` to access the element

Reply:
And then you can do like `within(getByRole('list')).getAllByRole('listitem')` to get all the accordion items

Reply:
sounds gooood